serverless.yml

```diff
@@ -19,42 +19,8 @@ provider:
   stage: ${opt:stage, 'prod'}
   region: ${opt:region, 'eu-central-1'}
   deploymentBucket:
-    name: ${env:SLS_DEPLOY_BUCKET}
+    name: ${env:SLS_DEPLOY_BUCKET, 'kaskadi-serverless-deployment-bucket'}
```

Comment:
isn't 'kaskadi-serverless-deployment-bucket' our default? are there cases were we dont want to use the default?

Reply:
I replied to the related issue some weeks ago but no I'm not defaulting to any bucket normally because I don't want to expose our bucket name to avoid potential attack vectors.

That's why I use environment variables instead but I'm not sure if that's necessary or if it's just being overly careful

Reply:
the deploy bucket is private anyways, right?

Reply:
Yep

Reply:
so then i think hardcoding it is fine